### ISSUE TITLE
Add Peers Graph

### ISSUE THREAD

**Issue by a1300** [GNYIO/gny-explorer#26]
## TODO
Please add a small **Peers** button on the top right corner. Which redirects to a `/peers` page

## Data to show
- How many nodes
- How many nodes are on a particular version (hint: `/api/system`)
- P2P graph (which node is connected to which other nodes) (`/api/peers`)
- Tabular list of every node with (hint: `/api/peers/info`)
  - `ip address`
  - version
  - height (show a `green` background-color when it is equal to the latest block, `red` background-color when it is different)

**Comment by xpgeng** [comment]
I tried to fetch all the data provided by System API and Peer API.  Some data I cannot find:

- nodes are on a particular version
  - `/api/system` returned data like

```json
{
  "success": true,
  "os": "linux_4.4.0-170-generic",
  "version": "1.0.28",
  "timestamp": 1591166965697,
  "lastBlock": {
    "height": "890403",
    "timestamp": 1591166960000,
    "behind": 0
 }
```
- P2P graph
  - Only connected nodes with current server can be obtained from `/api/peers`

```json
[
    {
      "id": {
        "id": "QmUmb9ejLz4SYqjJPUJL4MnU5SyrzyPomrt4CX4Zgf2sUH",
        "pubKey": "CAASpgIwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQCiEOnQwtIzxqGy7tpFs2QIYZeBV8wEVe4qTcBKJYWsMm9xC6NP+MTfpvVQgBuRhzBHAwmAWARDIYXNF9Yw/cO+TV0iUWu3DIJnvQTf2PcGxB95XotdAbYxORj4+31i8P/RdKjh0aYI7dz26BZA+pZXuQ/CAb277GqQH9Xky4oSZEaZvS1G12cE/Tw8w11NBpE1Wd61G6iez+aHOyqKYjQIDsPzT1L94Lj8JfvvGQX/O1p29pTiiDVX4rrJFwV0tmnnY9RL2+f/cPWCrnpQoh4YoF7HDFpJi3hTzS2rVpnrqMqp0dop3CZvtS4681tVBeuDENTFdsZTpyDFbW3m/hpzAgMBAAE="
      },
      "multiaddrs": [
        "/ip4/20.43.35.47/tcp/4097/ipfs/QmUmb9ejLz4SYqjJPUJL4MnU5SyrzyPomrt4CX4Zgf2sUH"
      ],
      "simple": {
        "host": "20.43.35.47",
        "port": 4097
      }
    },
    {
      "id": {
        "id": "QmXRsrJBxC65nTsb6c4GuidMdiqAw5TtkrGEFqpjbEJbPt",
        "pubKey": "CAASpgIwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQDHdR7jKzPCVFlgfXtNMth5UDMmfVSbsNLKuAjRsPkIWHCrOQEd78qRJi8MP1RebXjKd0PXUYLFQxelsmJ26YOeuUsoyKG5x1zYgPE1aOhyz6AaJCBM2EqIj+b4DHgwmgScfUWPLGjE++cCfQqzi10cKYKvDNZJOKnlh4KvH1Wj3a/Blbrmb4CIs8ha7VC0kxuKwjW352DC1rIkp4ParyoAcb6uKSrTaaV+FiyaPcwBq62v8jMHpdvGfzokWTIrJVEWgOV//R83OD40NrquhbNvrbh2fsjJuCyke7i6i8tdvXO2qc0feartuUZKCbCDgRF9P/LTj9sYTVc83KdKaoWVAgMBAAE="
      },
      "multiaddrs": [
        "/ip4/52.174.62.219/tcp/4097/ipfs/QmXRsrJBxC65nTsb6c4GuidMdiqAw5TtkrGEFqpjbEJbPt"
      ],
      "simple": {
        "host": "52.174.62.219",
        "port": 4097
      }
    },
...
```
- Peers information from `/api/peers/info`

```json
{
  "success": true,
  "id": "QmNT5ZNU8Nf9shpuz45phNHimUnsNZRj35B3ucSE3iKCk5",
  "multiaddrs": [
    "/ip4/45.76.215.117/tcp/4097/ipfs/QmNT5ZNU8Nf9shpuz45phNHimUnsNZRj35B3ucSE3iKCk5"
  ],
  "publicIp": "45.76.215.117",
  "address": "0.0.0.0"
}
```

Therefore, so far we maybe need more APIs to provide more detail information so that all the features above can be shown.

**Comment by a1300** [comment]
Closed by PR: #34
